projects/halide/project.yaml

```diff
@@ -0,0 +1,8 @@
+homepage: "https://halide-lang.org/"
+language: c++
+# TODO: Find primary contact
+primary_contact: ""
```

Comment:
@alexreinking would it make sense for you to be the primary contact for this repo? if so do you have a google/gmail or github associated email that I can put here?

Reply:
Probably better to put @steven-johnson as OSS-Fuzz is a Google project and he works at Google.

Reply:
If you would like I can add you to the auto_ccs instead and that'll give you access to the dashboard/stats/bug tracker etc. I think the primary contact has a few more privileges, on top of that but I'm unsure.

Reply:
> Probably better to put @steven-johnson as OSS-Fuzz is a Google project and he works at Google.

SGTM

Reply:
Done

Reply:
> If you would like I can add you to the auto_ccs instead and that'll give you access to the dashboard/stats/bug tracker etc. I think the primary contact has a few more privileges, on top of that but I'm unsure.

That would be good!

Reply:
Easy, should I use your @gmail from the git log or a different email. Note that it does need to be a gmail account for you to get full access to the bug tracker/dashboard.

Reply:
Yeah, `alex.reinking@gmail.com` is the best address

Reply:
Done